machina/core/validators.py

```diff
@@ -11,3 +17,34 @@ def __call__(self, value):
             # performed.
             return
         super().__call__(value)
+
+
+@deconstructible
+class MarkupMaxLengthValidator:
+    """
+    Validates the max length of an input written in the markup language defined in the settings.
+
+    This is a proxy that use the implementation defined in MACHINA_MARKUP_MAX_LENGTH_VALIDATOR
+    setting.
+    """
+
+    def __init__(self, limit_value):
+        self.validator = self._get_markup_maxlength_validator(limit_value)
+
+    def __call__(self, value):
+        self.validator(value)
+
+    @staticmethod
+    def _get_markup_maxlength_validator(max_length):
+        dotted_path = machina_settings.MARKUP_MAX_LENGTH_VALIDATOR
+        try:
+            module, validator = dotted_path.rsplit('.', 1)
+            module, validator = smart_str(module), smart_str(validator)
+            validator = getattr(__import__(module, {}, {}, [validator]), validator)
+            return validator(max_length)
+        except ImportError as e:
+            raise ImproperlyConfigured(
+                _('Could not import MARKUP_MAX_LENGTH_VALIDATOR {}: {}').format(
+                    machina_settings.MARKUP_MAX_LENGTH_VALIDATOR, e
+                )
+            )
```

Comment:
All this could be done directly at the module level. Otherwise this import could be done multiple times. You can probably the exact same approach as the one used for the [MarkupTextFieldWidget](https://github.com/ellmetha/django-machina/blob/master/machina/models/fields.py#L39).

That way the configured validator will be imported at the module level and then you can simply use it in this "validator proxy". 😉